modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -196,7 +199,7 @@ TEST_P(Test_ONNX_layers, Eltwise3D)
     throw SkipTestException("Test is enabled starts from 2019R1");
 #endif
     if (backend == DNN_BACKEND_INFERENCE_ENGINE && target != DNN_TARGET_CPU)
-        throw SkipTestException("Only CPU on DLIE backend is supported");
+        throw SkipTestException("Only CPU on DLIE backend, and CUDA is supported");
```

Comment:
In case of `backend == DNN_BACKEND_INFERENCE_ENGINE` check CUDA addition is not needed.